get_weather.py

```diff
@@ -0,0 +1,17 @@
+from decimal import *
+from datetime import date, timedelta, datetime
+import formatting_Utils
+import get_weather_Utils
+
+# Gets today's date and makes it a string
+today = date.today()
+today_str = formatting_Utils.maketimestring(today)
+
+#Gets the city Name and ID from Database (here 0), will be called in a while/for to get all cities
+city = get_weather_Utils.get_location(0)[0]
+city_id = get_weather_Utils.get_location(0)[1]
```

Comment:
why do you call get_location twice. Better save it to a variable

Reply:
Done. Getting a list of dicts back! 
Great Idea!